app/main.py

```diff
@@ -1,2 +1,74 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+    key_hash: int
+
+
 class Dictionary:
-    pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3
```

Comment:
The `THRESHOLD` is defined as `2 / 3`, which results in a float value of approximately 0.6667. If the intention was to use a fraction, it should be explicitly defined as a float, e.g., `0.67` or `0.6667`.